tst/standard/isorms.tst

```diff
@@ -566,17 +566,110 @@ true
+gap> R := ReesMatrixSemigroup(g, mat);;
+gap> S := ReesMatrixSemigroup(Group((1, 2)), [[()]]);;
+gap> auto := IdentityMapping(g);;
+gap> g_elms_list := [(), (1, 3), (), (), ()];;
+gap> RMSIsoByTriple(R, S, [(), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple:
+<R1> and <R2> are not isomorphic,
+gap> RMSIsoByTriple(R, R, [42, auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should be a permutation,
+gap> RMSIsoByTriple(R, R, [(1, 7), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should be a permutation on [1 .. 5],
+gap> RMSIsoByTriple(R, R, [(1, 4), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[1] should not map columns to rows,
+gap> RMSIsoByTriple(R, R, [(), fail, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[2] should be an isomorphism from
+the underlying group of <R1> to that of <R2>,
+gap> RMSIsoByTriple(R, R, [(), auto, [42, 43, 44]]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] should only contain elements from the underlying group of <R2>,
+gap> RMSIsoByTriple(R, R, [(), auto, [(), (), ()]]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] should have length equal to
+the number of rows and columns of <R1>,
+gap> RMSIsoByTriple(R, R, [(), auto, g_elms_list]);
+Error, Semigroups: RMSIsoByTriple: usage,
+<triple>[3] does not define an isomorphism,
+gap> g_elms_list := [(), (), (), (), ()];;
+gap> RMSIsoByTriple(R, R, [(), auto, g_elms_list]);
+((), IdentityMapping( SymmetricGroup( [ 1 .. 4 ] ) ), [ (), (), (), (), () ])
+
+#T# Errors in checked version of RZMSIsoByTriple
+gap> g := SymmetricGroup(3);;
+gap> mat := [[0, 0, (1, 3)], [(1, 2, 3), (), (2, 3)], [0, 0, ()]];;
+gap> R := ReesZeroMatrixSemigroup(g, mat);;
+gap> S := ReesZeroMatrixSemigroup(Group((1, 2)), [[()]]);;
+gap> auto := IdentityMapping(g);;
+gap> g_elms_list := [(), (1, 3), (), (), (), ()];;
+gap> RZMSIsoByTriple(R, S, [(), auto, g_elms_list]);
+Error, Semigroups: RZMSIsoByTriple:
+<R1> and <R2> are not isomorphic,
+gap> RZMSIsoByTriple(R, R, [42, auto, g_elms_list]);
+Error, Semigroups: RZMSIsoByTriple: usage,
+<triple>[1] should be a permutation,
+gap> RZMSIsoByTriple(R, R, [(1, 3), auto, g_elms_list]);
+Error, Semigroups: RZMSIsoByTriple: usage,
+<triple>[1] should act as an isomorphism from
+the graph of <R1> to the graph of <R2>,
+gap> RZMSIsoByTriple(R, R, [(), fail, g_elms_list]);
+Error, Semigroups: RZMSIsoByTriple: usage,
+<triple>[2] should be an isomorphism from
+the underlying group of <R1> to that of <R2>,
+gap> RZMSIsoByTriple(R, R, [(), auto, [42, 43, 44]]);
+Error, Semigroups: RZMSIsoByTriple: usage,
+<triple>[3] should only contain elements from the underlying group of <R2>,
+gap> RZMSIsoByTriple(R, R, [(), auto, [(), (), ()]]);
+Error, Semigroups: RZMSIsoByTriple: usage,
+<triple>[3] should have length equal to
+the number of rows and columns of <R1>,
+gap> RZMSIsoByTriple(R, R, [(), auto, g_elms_list]);
```

Comment:
And here can you create the same thing using the NC version of the function, and then check that it is not an isomorphism using the `BruteForceIsoCheck`?